### PR TITLE
User manipulation

### PR DESCRIPTION
**Pull request by jeffklassen**
- User Registration
- User creation by an admin
- User edit by an admin
- Profile edit

### CODE REVIEW

**Comment by mnmercer** [comment]
I'm not going to be able to get to this until Sunday, but I'll go through it as soon as I can.

File: client/src/App/components/users/createEdit/CreateEditForm.jsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        else {
+            return null;
+        }
+    };
+    return (
+        <div className="container" style={createEditFormStyle}>
+            <Well>
+                <form name="createEditForm" className="form-horizontal" onSubmit={handleSubmit}>
+                    <legend>{labels.title}</legend>
+                    <InputRow {...spacing} label="Username (Email)">
```
Comment (BKreisel) [inline client/src/App/components/users/createEdit/CreateEditForm.jsx]: Ooh. Spread Syntax. That is some EMCA Ninja Action 🔥

**Comment by BKreisel** [comment]
Slight UI bug where if you change a user first/last name while logged in as that user. The nav bar will not be updated with that users new first/last name.

**Comment by jeffklassen** [comment]
Hmm -- that's a good point. I solved that problem when you edit your own
profile.

I'll make a quick fix and re-deploy.

**Comment by jeffklassen** [comment]
@mnmercer We are merging this. In general, I think one person approving a Pull Request is enough. @BKreisel approved, so here we go!

**Pull request merged**